src/lib/containers/GenericCard.tsx

```diff
@@ -529,23 +561,33 @@ export default class GenericCard extends React.Component<
     )
     const values: string[][] = []
     const { secondaryLabels = [] } = genericCardSchemaDefined
+    const isView = table && !isTableEntity(table)
```

Comment:
Rendering an entity under the Table interface, but it's not a TableEntity.  It must be some type of Entity View.